examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>Can you wrap ds in a display(ds)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openradar/xradar/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>